challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map{|animal| animal.capitalize }
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  input.reject{|animal| animal != "dog"}
```

Comment:
consider `input.select{|animal| animal == "dog"}` to rephrase the double negative positively.